src/scripts/main.js

```diff
@@ -1,3 +1,207 @@
 'use strict';
 
-// write your code here
+const start = document.querySelector('.start');
```

Comment:
```suggestion
const startButton = document.querySelector('.start');
```
The name `start` implies that it's a function.